data/DaduDev.json

```diff
@@ -2,15 +2,24 @@
   "name": "Shaik Dadapeer",
   "type": "public",
   "bio": "Computer Science student just starting out with open source and soon be landing on a remote job",
+  "tags": [
+    "Open Source",
+    "Javascript",
+    "Java",
+    "React",
+    "Python",
+    "Docker",
+    "Kubernetes"
+  ],
   "links": [
     {
       "name": "Follow me on Twitter",
-      "url": "https://twitter.com/Dadudev1",
+      "url": "https://twitter.com/Dadu_Dev_",
```

Comment:
Lol, love your banner. <3

Reply:
> Lol, love your banner. <3

Thank you, I can make something like that for yourself too!